pokedex/simulator/difftest/src/spike.rs

```diff
@@ -74,6 +24,21 @@ impl DerefMut for SpikeLog {
 }
 
 impl SpikeLog {
+    pub fn parse_from(log: impl AsRef<str>) -> SpikeLog {
```

Comment:
For a complex function, I suggest remove unnecessary generics. Simply take a `&str` here.